src/containers/Penjualan/Table/TablePenjualan.tsx

```diff
@@ -45,19 +57,19 @@ const TablePenjualan = (props: handleShowType) => {
     setShowEdit(false)
   }
 
-  useEffect(() => {
-    axios.get('http://localhost:4000/nota-penjualan', config).then((response) => {
-      setData(response.data)
-    })
-  }, [])
-
   let token = cookies.token
   let config = {
     headers: {
       Authorization: 'Bearer ' + token,
     },
   }
 
+  useEffect(() => {
+    axios.get('http://localhost:4000/nota-penjualan', config).then((response) => {
+      setData(response.data)
+    })
+  }, [])
+
   const handleDelete = (id: string) => {
     axios
       .delete('http://localhost:4000/nota-penjualan/' + id, config)
```

Comment:
create env for this base api url http://localhost:4000/, dont hard code

Reply:
done get api from env